### ISSUE TITLE
Create an Accessibility Standard

### ISSUE THREAD

**Issue by DrAwesome4333** [DrAwesome4333/ap-tracker#30]
As of now, the app has had little to no thought put into accessibility,
Standards related to that should be created.

Accessibility needs that should be considered:
- Color blindness & Contrast
- Keyboard Navigation
- Readability
- Photo sensitivity
- Alt text
